CONTRIBUTING.md

```diff
@@ -13,13 +13,22 @@ change. For instance:
 
 ## Changelog
 
+### Overview
```

Comment:
Should `CHANGELOG.md` and `CHANGELOG-API.md` also mention each other at the top? Something like *"if you are looking for developer-facing changes, check out the changelog at `CHANGELOG-API.md`"*

Reply:
Good idea. I'll add this.